code/datasets.py

```diff
@@ -0,0 +1,25 @@
+
```

Comment:
This looks like that is loading the old format , we decided to go with csv format. please update this code to read a csv file 
category,word1,word2,word3,word4